packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -360,8 +376,18 @@ export class AppManagementClient implements DeveloperPlatformClient {
     }
   }
 
-  async devStoresForOrg(_orgId: string): Promise<OrganizationStore[]> {
-    return []
+  // we are returning OrganizationStore type here because we want to keep types consistent btwn
+  // partners-client and app-management-client. Since we need transferDisabled and convertableToPartnerTest values
+  // from the Partners OrganizationStore schema, we will return this type for now
+  async devStoresForOrg(orgId: string): Promise<OrganizationStore[]> {
+    const storesResult = await businessPlatformOrganizationsRequest<ListAppDevStoresQuery>(
+      ListAppDevStores,
+      await this.businessPlatformToken(),
+      orgId,
+    )
+    const shopArray = storesResult?.organization?.properties?.edges.map((value) => value.node as ShopNode) ?? null
```

Comment:
For 1, unfortunately I can't get around that because of the auto-generated type system. For some reason in the BP schema, these values can have a `null` type, even though technically it shouldn't.. 

For 2, great point! I've updated :)